From fda4ba37ece7c9c24602581a13c7c64db158dfa7 Mon Sep 17 00:00:00 2001
From: rofl0r <retnyg@gmx.net>
Date: Thu, 4 Jun 2015 21:03:25 +0100
Subject: [PATCH] Revert "fix makefile link order"

This reverts commit 377b474b40bb34fde018301294e02850dac928fe.

There still are problems with the link order.
Revert to the known-good version.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 52c6a88..58800f3 100644
--- a/Makefile
+++ b/Makefile
@@ -82,7 +82,7 @@ src/version.o: src/version.h
 	$(CC) $(CPPFLAGS) $(CFLAGS) $(CFLAGS_MAIN) $(INC) $(PIC) -c -o $@ $<
 
 $(LDSO_PATHNAME): $(LOBJS)
-	$(CC) -shared $(LDFLAGS) $(LD_SET_SONAME)$(LDSO_PATHNAME) -lpthread $(LIBDL) -o $@ $(LOBJS)
+	$(CC) -shared -lpthread $(LIBDL) $(LDFLAGS) $(LD_SET_SONAME)$(LDSO_PATHNAME) -o $@ $(LOBJS)
 
 $(ALL_TOOLS): $(OBJS)
 	$(CC) $(LDFLAGS) src/main.o src/common.o -o $(PXCHAINS)
